.github/workflows/build_test.yml

```
@@ -65,40 +58,11 @@ jobs:
-      - name: Build package
-        run: |
-          pixi run pack-wasm
-          mkdir -p package
-          cp vegafusion-wasm/pkg/vegafusion-wasm-*.tgz package/
-          echo "timestamp: $(date)" > package/info.txt
-      - name: Upload artifacts
-        uses: actions/upload-artifact@v4.4.3
-        with:
-          name: vegafusion-wasm-package
-          path: package
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)